applications/mne_scan/plugins/brainflowboard/README.md

```diff
@@ -0,0 +1,29 @@
+# BrainFlow Plugin
```

Comment:
and this file looks a little outdated, in my latest commits its removed and replaced by doc page. We can keep readme as well but lets make it like page from docs

Reply:
ups, yes you are right. The README.md should be removed from PR.

Reply:
removed